angular/projects/admin/src/app/services/document.service.ts

```diff
@@ -51,6 +52,17 @@ export class DocumentService {
       throw new Error('Document ID must be provided as an attribute when updating an document.');
     }
 
+    // Convert moment data to firestore timestamp
+    const data = document.data;
+    for (const key in data) {
+      if (moment.isMoment(data[key])) {
+        const toDate = moment(data[key]).toDate();
+        const toTimestamp = firebase.firestore.Timestamp.fromDate(toDate);
```

Comment:
Move this handling closer to the form element that provides a value. Optimally the date-time picker returns a Timestamp. But if it is not possible, then the form should be the only one knowing about the `moment` data type. 
Let the form or the form component change the value to `Timestamp` before passing to the service.

Reply:
Resolve on this commit https://github.com/oddbit/tanam/pull/56/commits/f103dd165b0bdeafa258cf7b7c488043900b7d50